### DISCUSSION TITLE
Possible to create rest api with remix ?

### DISCUSSION THREAD

**Discussion by hakimLyon** [remix-run/remix#4557] [Q&A] [answered]
Hi  😄 ,

I would like to make an API call from my mobile app to post data to my remix app.

That's possible ? 

![Capture d’écran du 2022-11-09 20-30-35](https://user-images.githubusercontent.com/64654633/200936239-2995dab8-742d-4a11-8d9b-0caac6bdc7d7.png)

![Capture d’écran du 2022-11-09 20-31-02](https://user-images.githubusercontent.com/64654633/200936260-0409ce6a-c209-4c30-bc3f-c58ad55eb7d7.png)

**Comment by kiliman** [comment, marked as answer]
Yes, you can create a resource route as your mobile API.

In your example, I think it's failing because you're trying to send request.body (a stream) as JSON.

Instead send an actual payload. Like:
```
export async function action({request}: ActionArgs) {
  return json(await request.json())
}
```

**Reply by hakimLyon** [reply]
@kiliman  😄 Haha cool , thank you so much 💯
